kube_log_watcher/main.py

```diff
@@ -47,8 +49,8 @@ def get_container_label_value(config, label) -> str:
         io.kubernetes.pod.name
     """
     labels = config['Config']['Labels']
-    for l, val in labels.items():
-        if l.endswith(label):
```

Comment:
How is `i` better than `l`? 😄 I would understand `k`...
I don't really care either way, just curious why you bothered to change this in this PR.

Reply:
Flake (new version? there wasn't this problem before) in general doesn't like short names, but it's fine with `i`. `label` is already in use here, I didn't come up with better name